overlay/sbin/k3s-upgrade

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+if [ $(whoami) != "root" ]
+then
+    echo "This script must be run as root."
+    exit 1
+fi
+
+PROC=$(uname -m)
+
+# Take a parameter of the version number (i.e. v0.4.0) if it is given, otherwise use latest
+if [ -z $K3S_VERSION ]
+then
+    K3S_VERSION=$(curl -sL api.github.com/repos/rancher/k3s/releases/latest | jq .tag_name -r)
+fi
+
+if [ $PROC == "x86_64" ]
+then
+    ARCH="amd64"
```

Comment:
On AMD64, the k3s release binary is simply called 'k3s' with no arch suffix. In this case we should say something like.

# Special case for amd64 - k3s release has no arch suffix
ARCH=""